HyperV/scripts/config.ps1

```diff
@@ -20,5 +20,6 @@ $pythonExec = "$pythonDir\python.exe"
 $openstackLogs="$openstackDir\Log"
 $remoteLogs="\\"+$devstackIP+"\openstack\logs"
 $remoteConfigs="\\"+$devstackIP+"\openstack\config"
+$eventlogPath="C:\OpenStack\Logs\Eventlog"
```

Comment:
$openstackLogs="$openstackDir\Log"

Why we use here "c:\openstack\logs" (with the S at the end) and not relying on the pre-defined variables above?